src/core/CMakeLists.txt

```diff
@@ -240,6 +250,9 @@ add_library(
   ${SERVER_SRCS} ${SERVER_HDRS}
 )
 add_dependencies(server-library proto-library)
+if(${TRITON_ENABLE_GPU})
+  add_dependencies(server-library gather_kernel)
```

Comment:
can you use "target-link-libraries server-library gather-kernel"
and then I don't think you need the change in src/servers/CMakeLists.txt

Reply:
It doesn't work so I still link it in `tritonserver` target. But I remove the dependency here because the symbol will be resolved in `tritonserver`